src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  );
+}
```

Comment:
for future: use arrow functions to make your code prettier and shorter 
```suggestion
export const Sum = ({ a = 0, b = 0 })  => (
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
);
```